pkg/pod/network.go

```diff
@@ -13,3 +13,14 @@ func StaticIPAnnotation(name string, ipAddr []string) []*multus.NetworkSelection
 		},
 	}
 }
+
+// StaticIPAnnotationWithInterfaceAndNamespace defines static ip address, interface name and namespace
+// network annotation for pod object.
+func StaticIPAnnotationWithInterfaceAndNamespace(
+	name, namespace, intName string, ipAddr []string) []*multus.NetworkSelectionElement {
```

Comment:
can you please define each parameter on a separate line

Reply:
According to convention from README it's OK

```
func Function(
    argInt1, argInt2 int, argString1, argString2 string, argSlice1, argSlice2 []string) {
	
}
```